go/vt/wrangler/schema.go

```diff
@@ -564,7 +564,7 @@ func (wr *Wrangler) applySqlShard(ctx context.Context, tabletInfo *topo.TabletIn
 	ctx, cancel := context.WithTimeout(ctx, 30*time.Second)
 	defer cancel()
 	// Need to make sure that we enable binlog, since we're only applying the statement on masters.
-	_, err = wr.tmc.ExecuteFetchAsDba(ctx, tabletInfo, filledChange, 0, false, false)
+	_, err = wr.tmc.ExecuteFetchAsDba(ctx, tabletInfo, filledChange, 0, false, false, true)
```

Comment:
We could pass in the boolean to applySqlShard, and in the for loop above, change code to:
for i, sqlLine := range createSql {
    err = wr.applySqlShard(ctx, tabletInfo, sqlLine, i == len(createSql)-1)
So we only reload schema for the last one?

(if there is an error, we're in trouble anyway, not having reloaded the schema is the last of our worries)